package.json

```diff
@@ -293,6 +276,24 @@
       "hash": "QmcBWojPoNh4qm7zvv4qiepvCnnc7ALS9qcp7TNwwxT1gT",
       "name": "go.uuid",
       "version": "1.1.0"
+    },
+    {
+      "author": "whyrusleeping",
+      "hash": "QmYkxWtBWqkj6SWCiFN2a3xnnk5womAQYd9Kh68LJJR7jz",
+      "name": "go-ws-transport",
```

Comment:
do we have to depend on a transport?

Reply:
It's a default transport so yes unless we want to make it not a default transport. Note: we used to depend on it in go-libp2p-swarm but I moved all default transport construction here.